broker-daemon/broker-rpc/admin-service/register.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Register the publicKey with the Relayer
+ *
+ * @param {GrpcUnaryMethod~request} request - request object
+ * @param {RelayerClient} request.relayer - grpc Client for interacting with the Relayer
+ * @param {Object} request.logger
+ * @param {Object} responses
+ * @param {function} responses.RegisterResponse - constructor for RegisterResponse messages
+ * @return {RegisterResponse}
+ */
+async function register ({ relayer, logger }, { RegisterResponse }) {
+  const publicKey = relayer.identity.pubKeyBase64
+
+  // Currently we don't do anything with this entityId but we will need it in the future
+  const { entityId } = await relayer.adminService.register({ publicKey })
+
```

Comment:
might be worthwhile to add a log here saying "successfully registered with relayer and include the entity id